From: juhosg <juhosg@3c298f89-4303-0410-b956-a3cf2f4a3e73>
Date: Mon, 25 Mar 2013 06:40:00 +0000
Subject: ar71xx: add GPIO output select values for AR934x

Signed-off-by: Gabor Juhos <juhosg@openwrt.org>

git-svn-id: svn://svn.openwrt.org/openwrt/trunk@36121 3c298f89-4303-0410-b956-a3cf2f4a3e73

Conflicts:
	target/linux/ar71xx/patches-3.8/601-MIPS-ath79-add-more-register-defines.patch

diff --git a/target/linux/ar71xx/patches-3.3/601-MIPS-ath79-add-more-register-defines.patch b/target/linux/ar71xx/patches-3.3/601-MIPS-ath79-add-more-register-defines.patch
index 8411d58..12522c1 100644
--- a/target/linux/ar71xx/patches-3.3/601-MIPS-ath79-add-more-register-defines.patch
+++ b/target/linux/ar71xx/patches-3.3/601-MIPS-ath79-add-more-register-defines.patch
@@ -166,7 +166,7 @@
  #define AR934X_GPIO_REG_FUNC		0x6c
  
  #define AR71XX_GPIO_COUNT		16
-@@ -550,4 +618,139 @@
+@@ -550,4 +618,144 @@
  #define AR934X_SRIF_DPLL2_OUTDIV_SHIFT	13
  #define AR934X_SRIF_DPLL2_OUTDIV_MASK	0x7
  
@@ -237,7 +237,12 @@
 +#define AR934X_GPIO_FUNC_CLK_OBS0_EN		BIT(2)
 +#define AR934X_GPIO_FUNC_JTAG_DISABLE		BIT(1)
 +
-+#define AR934X_GPIO_OUT_GPIO		0x00
++#define AR934X_GPIO_OUT_GPIO          0
++#define AR934X_GPIO_OUT_LED_LINK0     41
++#define AR934X_GPIO_OUT_LED_LINK1     42
++#define AR934X_GPIO_OUT_LED_LINK2     43
++#define AR934X_GPIO_OUT_LED_LINK3     44
++#define AR934X_GPIO_OUT_LED_LINK4     45
 +
 +/*
 + * MII_CTRL block
